game/state/gamestate.cpp

```diff
@@ -654,31 +654,37 @@ void GameState::fillPlayerStartingProperty()
 	    auto v = current_city->placeVehicle(*this, {this, type}, this->getPlayer(), {this, bld});
 	    v->homeBuilding = v->currentBuilding;
 	}*/
+
 	for (auto &pair : this->initial_vehicles)
 	{
 		auto v = current_city->createVehicle(*this, pair.first, this->getPlayer(), {this, bld});
 		v->homeBuilding = v->currentBuilding;
 		for (auto &eq : pair.second)
 		{
-			v->addEquipment(*this, eq);
+			auto device = v->addEquipment(*this, eq);
```

Comment:
This is setting all the initial vehicle ammo to full? I've noticed that before, good to have it fixed - but it should probably be a different change?

Reply:
Yes, it's been a bug in OA for a while
Wasn't sure if to separate or not as it relates to the weapon this PR is changing for burst mode
If you want it separate i can do that?
I have to change the burst from 10 to 3 anyway